resources/views/service/registration.blade.php

```diff
@@ -0,0 +1,100 @@
+@extends('layouts.service_master')
+
+@section('title', 'Dashboard')
+
+@section('content')
+    <div class="content">
+        <form>
+            <div class="col">
+                <div>
+                    <h2>
+                        <label for="pri_carer">
+                            <i class="fa fa-user" aria-hidden="true"></i> Main Carer's full name:
+                        </label>
+                    </h2>
+                    <input id="pri_carer" type="text">
+                </div>
+                <div class="add">
+                    <h2>
+                        <label for="sec_carers">
+                            <i class="fa fa-user" aria-hidden="true"></i> Other voucher collectors:
+                        </label>
+                    </h2>
+                    <input id="sec_carers" type="text">
+                    <button name="add-collector">
+                        <i class="fa fa-plus" aria-hidden="true"></i>
+                    </button>
+                </div>
+                <div class="collectors">
+                    <p>These people may collect vouchers:</p>
+                </div>
+            </div>
+            <div class="col">
+                <div class="add">
+                    <h2>
+                        <i class="fa fa-user-plus" aria-hidden="true"></i>
+                        Add Children <span><i class="fa fa-info-circle" aria-hidden="true"></i></span>
+                    </h2>
+                    <h3>
+                        <label for="birth_month">Month + Year of birth (or due date for pregnancy)</label>
+                    </h3>
+                    <input id="birth_month" type="month" min="1998-01">
+                    <button name="add-collector">
+                        <i class="fa fa-plus" aria-hidden="true"></i>
+                    </button>
+                </div>
+                <div>
+                    <h2>Children signed up:</h2>
+                    <table>
+                        <tr>
+                            <td>Age</td>
+                            <td>Month/Year</td>
+                            <td>Info</td>
+                        </tr>
+                        <tr>
+                            <td>7</td>
```

Comment:
Stubs?

Reply:
yup, that's going to be flattened by a vue component.